provider/core/src/hello_world.rs

```diff
@@ -111,12 +111,12 @@ impl HelloWorldProvider {
         ("de-AT", "", "Servus Welt"),
         ("el", "", "Καλημέρα κόσμε"),
         ("en", "", "Hello World"),
-        ("en-001", "", "Hello from 🗺️"),            // WORLD
-        ("en-002", "", "Hello from 🌍"),           // AFRICA
-        ("en-019", "", "Hello from 🌎"),           // AMERICAS
-        ("en-142", "", "Hello from 🌏"),           // ASIA
-        ("en-GB", "", "Hello from 🇬🇧"),            // GREAT BRITAIN
-        ("en-GB-u-sd-gbeng", "", "Hello from 🏴󠁧󠁢󠁥󠁮󠁧󠁿"), // ENGLAND
+        ("en-001", "", "Hello from 🗺️"),        // WORLD
+        ("en-002", "", "Hello from 🌍"),       // AFRICA
+        ("en-019", "", "Hello from 🌎"),       // AMERICAS
+        ("en-142", "", "Hello from 🌏"),       // ASIA
+        ("en-GB", "", "Hello from 🇬🇧"),        // GREAT BRITAIN
+        ("en-GB", "england", "Hello from 🏴󠁧󠁢󠁥󠁮󠁧󠁿"), // ENGLAND
```

Comment:
Issue: `england` should remain a subdivision, not a key attribute... please at least add a TODO(#4413)

Reply:
This is as designed